fork_choice/safe-block.md

```diff
@@ -0,0 +1,46 @@
+# Fork Choice -- Safe Block
```

Comment:
Note that this file is cross forks. Maybe note that it is only valid post-Bellatrix.

Reply:
In general, safe block algorithm can be implemented with `Phase0` structures. I would add a note that `get_safe_execution_payload_hash` helper is based on `Bellatrix`